src/main/java/pages/UserAccountPage.java

```diff
@@ -10,6 +10,15 @@ public class UserAccountPage extends BasePage<UserAccountPage> {
     private static final By MY_ACCOUNT_HEADER = xpath("//*[@data-ui-id='page-title-wrapper']");
     private static final By MY_ACCOUNT_EMAIL = xpath("//p[@class='dashboard-info-block__email']");
     private static final By TEXT_OUT = xpath("//*[@class='nav items']/li[10]/a");
+    private static final String ACCOUNT_SETTING = "//*[@class='nav item'][4]/a";
+    private static final String FIRST_NAME_FIELD = "//input[@name='firstname']";
+    private static final String SUR_NAME_FIELD = "//input[@name='lastname']";
+    private static final String NOTIFICATION_CHECK_BOX_MAIL_LABEL = "//label[@for='gdpr-agreement-f56754fe-fa50-409a-9ec2-8706631db48c']";
+    private static final String NOTIFICATION_CHECK_BOX_PHONE_LABEL = "//label[@for='gdpr-agreement-acab0796-e6e9-41d1-bbd3-9af8c46e277f']";
+    private static final String NOTIFICATION_CHECK_BOX_MAIL_INPUT = "//input[@id='gdpr-agreement-f56754fe-fa50-409a-9ec2-8706631db48c']";
+    private static final String NOTIFICATION_CHECK_BOX_PHONE_INPUT = "//input[@id='gdpr-agreement-acab0796-e6e9-41d1-bbd3-9af8c46e277f']";
+    private static final String SAVE_CHANGE_BUTTON = "//button[@class='action save account-edit-info__button button button--regular button__primary']";
+    private static final By SUCCESS_MESSAGE = xpath("//div[@class='message-text' and contains(text(), 'Дані Вашого Облікового запису збережено')]");
```

Comment:
why use By when all other is string
```suggestion
    private static final String SUCCESS_MESSAGE ="//div[@class='message-text' and contains(text(), 'Дані Вашого Облікового запису збережено')]";
```